client/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:latest
```

Comment:
Didn't you mention this was not supposed to be here? Or are we keeping it for the (near) future possibility of containerizing everything?

Reply:
keeping for future work.